frontend/app/src/components/settings/data-security/DataManagement.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <v-row no-gutters>
+    <v-col>
+      <v-card>
+        <v-card-title>Manage Data</v-card-title>
+        <v-card-subtitle>
+          Allows you to purge data cached by rotki
+        </v-card-subtitle>
+        <v-form ref="form">
+          <v-card-text>
+            <h3>Exchange Data</h3>
+            <v-row align="center">
+              <v-col>
+                <v-select
+                  v-model="exchange"
+                  class="data-management__fields__exchange"
+                  :items="exchanges"
+                  :success-messages="
+                    exchangePurgeSuccess
+                      ? ['Cached data where successfully purged']
+                      : []
+                  "
+                  :error-messages="exchangePurgeError"
+                  label="Exchange"
+                >
+                  <template #item="{ item, attrs, on }">
+                    <v-list-item
+                      :class="`exchange__${item}`"
+                      v-bind="attrs"
+                      v-on="on"
+                    >
+                      {{ item }}
+                    </v-list-item>
+                  </template>
+                </v-select>
+              </v-col>
+              <v-col cols="auto">
+                <v-tooltip top>
+                  <template #activator="{ on, attrs }">
+                    <v-btn
+                      class="data-management__purge-exchange"
+                      icon
+                      :disabled="!!!exchange"
+                      v-bind="attrs"
+                      v-on="on"
+                      @click="confirmExchangePurge = true"
+                    >
+                      <v-icon>fa-trash</v-icon>
+                    </v-btn>
+                  </template>
+                  <span>Purge cached data for the selected exchange</span>
```

Comment:
I don't see a way to delete ALL cached data for all exchanges. There is an endpoint for this if you saw the docs. (just omit specific exchange name). Would it be hard to also add this option?